### ISSUE TITLE
Doesn't work with jQuery Slim Build

### ISSUE THREAD

**Issue by marcus-at-localhost** [mikejacobson/jquery-bootstrap-scrolling-tabs#77]
jQuery Slim comes without animation effects and as far as I can tell, this plugin relies on them.
Here is a non working example https://codepen.io/localhorst/pen/JjoNRxx

Obligatory question: Any chance this will become a vanilla js lib in the future (since bootstrap is moving away from it as well)? :)

**Comment by mikejacobson** [comment]
Correct, this plugin relies on jQuery animations. I originally built it back in the beforetimes when I needed to support IE8, so I couldn't use CSS transforms/transitions/animations.

I just started working on a fork of the plugin that's specifically for BS4 and uses jQuery Slim. If that comes together, the next step would be a vanilla js solution.

Thanks for the suggestion and the link to the vanilla js scroller.
